__docs__/wonder-blocks-pill/pill.stories.tsx

```diff
@@ -213,8 +213,13 @@ export const Variants: StoryComponentType = {
 export const WithTypography: StoryComponentType = () => (
     <Pill size="large">
         <BodySerif>
-            This is a {<Link href="#link">link example</Link>} inside the text
-            of a pill.
+            This is a{" "}
+            {
+                <Link href="#link" style={{fontFamily: "inherit"}}>
```

Comment:
Would it make sense to set the `fontFamily` for `Link` in the Classic theme to `inherit` so that all existing Links don't need to be updated to inherit font properties? Or is this more a one off? I'm wondering if there are similar use cases that exist across the platform!